content/faqs/html-lazy-not-working.md

```diff
@@ -10,6 +10,15 @@ The [Elm guide on Html.Lazy](https://guide.elm-lang.org/optimization/lazy.html)
 
 Here are some additional gotcha's that are helped by examples.
 
+<toc></toc>
+
+
+Don't want to remember these gotchas? You don't have to! Let `elm-review` remember for you:
```

Comment:
To note: this rule was a WIP, and I don't remember where it's at :sweat_smile:

Reply:
Oh right! Does the NRI one fare better right now then?

Reply:
That one is a lot more restrictive on how you define lazy functions (always top-level, which helps with some of the problems but not all), whereas mine tries to find problems rather than force you into a way of using lazy functions

Reply:
Okay, so perhaps mention both then? 🤔

Reply:
Sure. If you want to, help on my rule would be appreciated :D